frontend/src/AddQuestionPage.js

```diff
@@ -101,74 +118,164 @@ const AddQuestionPage = () => {
         });
     }
 
+    const translateQuestionButton = () =>
+    {
+        if(areQuestionsLoading)
+        {
+            return (
+                <Button variant="primary" disabled>
+                    <Spinner
```

Comment:
I would add a little bit more space between the `Spinner` and the `Loading...` text to make things less squished together.